apps/search/src/modules/configuration/configuration.router.ts

```diff
@@ -48,7 +48,7 @@ export const configurationRouter = router({
   setConnectionConfig: protectedClientProcedure
     .input(AppConfigurationSchema)
     .mutation(async ({ input, ctx }) => {
-      loggerContext.set("saleorApiUrl", ctx.saleorApiUrl);
+      loggerContext.set(ObservabilityAttributes.SALEOR_API_URL, ctx.saleorApiUrl);
```

Comment:
suggestion: Would be great to have this on every log

Reply:
In this PR I added it to places I saw this was missing based on recent errors